.github/workflows/azure-static-web-apps-ashy-river-0debb7803.yml

```diff
@@ -12,6 +12,13 @@ on:
 jobs:
   build_and_deploy_job:
     if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    permissions:
+      actions: read 
+      checks: write
+      contents: read
+      deployments: read
+      pull-requests: write
+      security-events: write

```

Comment:
Seems like this is the default recommendation but I don't think the token is used for security events for example. Can we reduce this to the minimum needed? From reading this workflow file it looks like it only needs to write back to PR comments

Reply:
Thanks @alfredodeza - Updated permissions for prs and actions only